docs/mkdocs.yml

```diff
@@ -7,43 +7,15 @@ docs_dir: content
 edit_uri: https://github.com/hedgedoc/hedgedoc/edit/master/docs/content/
 nav:
   - Home: index.md
-  - Installation:
-      - 'Manual Installation': setup/manual-setup.md
-      - 'Reverse Proxy': setup/reverse-proxy.md
-      - Docker: setup/docker.md
-      - Cloudron: setup/cloudron.md
-      - Heroku: setup/heroku.md
-      - LinuxServer: setup/docker-linuxserver.md
-      - Yunohost: setup/yunohost.md
```

Comment:
I would keep these

Reply:
Well, currently 2.0 does not support being installed in any useful way, so keeping any installation instructions is not very useful.

As soon as we have a way of combining frontend & backend etc., we can re-add installation instructions.

Reply:
okay